tests/cpp_check_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+zil_baseurl="https://raw.githubusercontent.com/Zilliqa/Zilliqa/master/src"
+
+files=(
+    "libUtils/DataConversion.cpp"
+    "libUtils/DataConversion.h"
+    "libCrypto/Schnorr.cpp"
+    "libCrypto/Schnorr.h"
+    "libCrypto/Sha2.h"
+    "common/Serializable.h"
+    )
+
+for file in ${files[@]}
+do
+    bname=`basename $file`
+    wget -qO /tmp/${bname} ${zil_baseurl}/${file}
+    if ! cmp -s /tmp/${bname} src/cpp/Zilliqa/${bname}
```

Comment:
This is another hack. You should compare version instead of diffing the file! Using tags/releases is better. The difference in line ending should cause enough headache. wget doesn't work like git!

Reply:
The answer to both the comments is that: this is just a warning that the Schnorr sources need to be updated. There is manual intervention needed after this, to take a call whether `master` has the best version or not and whether we want to update.